R/gefs.R

```diff
@@ -218,3 +217,14 @@ gefs_dimension_values <- function(dim, con = NULL, ...) {
   if (is.null(con)) con = gefs_CONNECT(...)
   con$dim[[dim]]$vals
 }
+
+#' Check that package is installed
+#' 
+#' @param x the package name
+check4pkg <- function(x) {
```

Comment:
can you remove this fxn?  it's defined in the `zzz.R` file, for utility fxns https://github.com/ropensci/rnoaa/blob/5f5c89ac1376faa745d7a60064ebb11b8f0d0b28/R/zzz.r#L153-L160